news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 Beta Is Ready To Be Tested!"
+subtitle: "You really should download it and test it"
+date:   2020-04-17 12:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/04/1.7.7_beta_banner.jpg
+icon: icon-leaf
+tags:
+ - 1.7.7.x
+ - version
+ - beta
+ - minor
+ - releases
+---
+
+Go test the first Beta version of PrestaShop 1.7.7.0!
+
+![Prestashop 1.7.7.0 BETA Release](/assets/images/2020/04/1.7.7_beta_banner.jpg)
+
+After more than one year in the making, we are really thrilled to announce that we are releasing the 1.7.7.0 beta. This version is even bigger than the 1.7.6.0 with 957 merged pull requests, against 600 for the 1.7.6.0, and we are still a month away from the final version. 
+
+This huge work has been possible thanks to the engagement of our great community, with more than 110 external contributors participating in the development of this version, fixing bugs, adding improvement to existing features or even developing new features for our merchants !
+In this version we covered essential topics such as order management. By the way, you will enjoy more efficiency in daily tasks with the [full redesign and new features of order pages](). And also more stability with the upgraded compatibility to PHP 7.3 and more flexibility with new hooks.
+ 
+This beta release is essential for you to start testing the upcoming next PrestaShop version before the release of the stable version in more or less a month. During this period, we strongly recommend checking that all your modules and themes are still working, otherwise you need to update them or open an issue. You must download, install and test this new version of PrestaShop and give your feedback as early as possible during the beta period. Remember [our article about minor release lifecycle](https://build.prestashop.com/news/ps17-minor-release-lifecycle/): the sooner a problem is identified, the sooner it will be fixed. 
```

Comment:
```suggestion
It is essential that you to start testing this beta version before the release of the stable version in about a month. During this beta period, we strongly recommend checking that all your modules and themes are working properly, especially if you develop them yourself. Please download, install and test this new version of PrestaShop and give us your feedback as early as possible. Remember [our article about minor release lifecycle](https://build.prestashop.com/news/ps17-minor-release-lifecycle/): the sooner a problem is identified, the sooner it can be fixed. 
```